modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/download/DownloadHelper.kt

```diff
@@ -11,18 +12,25 @@ import kotlinx.coroutines.runBlocking
 
 object DownloadHelper {
 
-    fun manuallyDownloadEpisodeNow(episode: BaseEpisode, from: String, downloadManager: DownloadManager, episodeManager: EpisodeManager, fireToast: Boolean = false) {
+    fun manuallyDownloadEpisodeNow(
+        episode: BaseEpisode,
+        from: String,
+        downloadManager: DownloadManager,
+        episodeManager: EpisodeManager,
+        fireToast: Boolean = false,
+        source: SourceView,
```

Comment:
Optional parameters should be last ones for easier calling.

Reply:
Nice catch! I've updated the code with this change